pkg/services/sqlstore/org_users.go

```diff
@@ -142,6 +142,68 @@ func GetOrgUsers(query *models.GetOrgUsersQuery) error {
 	return nil
 }
 
+func (ss *SQLStore) SearchOrgUsers(query *models.SearchOrgUsersQuery) error {
+	query.Result = models.SearchOrgUsersQueryResult{
+		OrgUsers: make([]*models.OrgUserDTO, 0),
+	}
+
+	sess := x.Table("org_user")
+	sess.Join("INNER", x.Dialect().Quote("user"), fmt.Sprintf("org_user.user_id=%s.id", x.Dialect().Quote("user")))
+
+	whereConditions := make([]string, 0)
+	whereParams := make([]interface{}, 0)
+
+	whereConditions = append(whereConditions, "org_user.org_id = ?")
+	whereParams = append(whereParams, query.OrgID)
```

Comment:
Makes for less code:
```suggestion
	whereConditions := []string{"org_user.org_id = ?"}
	whereParams := []interface{query.OrgID}

	whereConditions = append(whereConditions, "org_user.org_id = ?")
	whereParams = append(whereParams, query.OrgID)
```